src/main/interface/msp.c

```diff
@@ -1779,8 +1779,14 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             const uint8_t command = sbufReadU8(src);
             if (command) {
                 setArmingDisabled(ARMING_DISABLED_MSP);
+#ifdef USE_RUNAWAY_TAKEOFF
+                runawayTakeoffTemporaryDisable(false);
+#endif
             } else {
                 unsetArmingDisabled(ARMING_DISABLED_MSP);
+#ifdef USE_RUNAWAY_TAKEOFF
+                runawayTakeoffTemporaryDisable(true);   // temporarily disable runaway takeoff prevention
```

Comment:
The way this is done now we won't be able to use it in a safe way. We need to be able to use `unsetArmingDisabled(ARMING_DISABLED_MSP)` while disabling runaway takeoff prevention (when enabling the motors in configurator), and use it without disabling runaway takeoff prevention (when exiting the configurator).

The idea is to add an extra (optional) byte to the command that is a boolean for `runawayTakeoffTemporaryDisable()`, defaulting to `false`.